src/helpers/event.ts

```diff
@@ -131,25 +131,19 @@ export const convertEventActionToEventTreeNode = (event: EventAction): EventTree
 	};
 };
 
-export const getErrorEventTreeNode = (eventId: string): EventTreeNode => {
-	return {
-		type: ActionType.EVENT_TREE_NODE,
-		isUnknown: true,
-		eventId,
-		eventName: eventId,
-		eventType: '',
-		filtered: true,
-		parentId: null,
-		startTimestamp: {
-			nano: 0,
-			epochSecond: 0,
-		},
-		endTimestamp: {
-			nano: 0,
-			epochSecond: 0,
-		},
-		successful: false,
-	};
+export const unknownRoot: EventTreeNode = {
+	type: ActionType.EVENT_TREE_NODE,
+	isUnknown: true,
+	eventId: 'unknown-root',
+	eventName: 'Unknown Events',
+	eventType: 'eventTreeNode',
+	filtered: false,
```

Comment:
it looks like `filtered` is not used anywhere, maybe we should delete it?

Reply:
`filtered` is a required property in 'EventTreeNode'

Reply:
is it ? i can't find any reference to `.filtered` as far as i remember this property is from old api

Reply:
Ok, I deleted it